src/main/java/org/formation/model/Formation.java

```diff
@@ -114,4 +114,9 @@ public Integer getId() {
 	public void setId(Integer id) {
 		this.id = id;
 	}
+	
+	public boolean isFusionnee() {
+		if (this.statut == "Fusionnee") return true;
+		return false;
```

Comment:
C'est bien, mais prenez l'habitude de mettre :
return this.statut == "Fusionnee";